doc/classes/PropertyTweener.xml

```diff
@@ -43,6 +43,25 @@
 				[/codeblock]
 			</description>
 		</method>
+		<method name="set_custom_interpolator">
+			<return type="PropertyTweener" />
+			<param index="0" name="interpolator_method" type="Callable" />
+			<description>
+				Allows interpolating the value with a custom easing function. The provided [param interpolator_method] will be called with a value ranging from [code]0.0[/code] to [code]1.0[/code] and is expected to return a value within the same range (values outside the range can be used for overshoot). The return value of the method is then used for interpolation between initial and final value. Note that the parameter passed to the method is still a subject to the tweener's own easing.
```

Comment:
```suggestion
				Allows interpolating the value with a custom easing function. The provided [param interpolator_method] will be called with a value ranging from [code]0.0[/code] to [code]1.0[/code] and is expected to return a value within the same range (values outside the range can be used for overshoot). The return value of the method is then used for interpolation between initial and final value. Note that the parameter passed to the method is still subject to the tweener's own easing.
```
If you mean "is controlled by/affected by" the phrase is "X is subject to", "subject" here is an adjective, see [here](https://en.m.wiktionary.org/wiki/subject#Adjective)